plugins.txt

```diff
@@ -0,0 +1,18 @@
+ansible
+blueocean
+blueocean-i18n
+blueocean-config
+blueocean-events
+blueocean-web
+blueocean-commons
```

Comment:
I think these are too specific to your own project's needs, this container is used by many different projects so should be fairly generic